ray-operator/config/samples/ray-job.sample.yaml

```diff
@@ -102,7 +102,7 @@ spec:
   #         image: rayproject/ray:2.9.0
   #         # If Command is not specified, the correct command will be supplied at runtime using the RayJob spec `entrypoint` field.
   #         # Specifying Command is not recommended.
```

Comment:
I'm a bit confused, are we recommending the user to specify `command` now in all cases or only in some cases? Either way, we should include an accompanying doc update for the new env var.  This YAML comment alone might be easily overlooked by a user.

Reply:
If `entrypoint` works, users should use `entrypoint`. If `entrypoint` doesn't work, for example, if the `ray` binary is not in the PATH, they can use `command`. I think the comment is fine. Several users use `submitterPodTemplate`, but no one set the `command` by themselves. For more details, they can't do that before #1852 was merged.

Reply:
> Either way, we should include an accompanying doc update for the new env var.

I ignored this. I will update the documentation on the Ray website.